.circleci/config.yml

```diff
@@ -9,123 +9,131 @@ executors:
+              --provenance=true \
+              --push \
+              -t 017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:${CIRCLE_SHA1}-amd64 \
+              .
+  publish_docker_to_ecr:
+    executor: docker-executor
+    steps:
+      - checkout
+      - setup-docker-ecr-login
       - run:
           name: Push Multi-Arch Docker Image
           command: |
             VERSION_FROM_GIT=$(git describe --tags --match='v[0-9]*' --always)
             VERSION_FROM_GIT=${VERSION_FROM_GIT#'v'}
             
-            docker buildx build \
-              --platform linux/amd64,linux/arm64 \
-              --sbom=true \
-              --provenance=true \
-              --push \
+            docker buildx imagetools create \
               -t 017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:$CIRCLE_SHA1 \
               -t 017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:$VERSION_FROM_GIT \
-              .
+              017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:${CIRCLE_SHA1}-amd64 \
+              017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:${CIRCLE_SHA1}-arm64
 
-  publish-multiarch-to-dockerhub:
+  publish_docker_to_dockerhub:
     executor: docker-executor
-    resource_class: xlarge
     steps:
-      - checkout
-      - setup_remote_docker:
-          privileged: true  # Required for Buildx
-          docker_layer_caching: true
+      - setup-docker-ecr-login
       - run:
-          no_output_timeout: 30m
-          name: Set up Docker Buildx
+          name: Pull from ECR and Push
           command: |
-            docker buildx create --use --name multiarch_builder
-            docker buildx inspect --bootstrap  # Ensures everything is set up correctly
+            docker pull 017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:$CIRCLE_SHA1-arm64
+            docker tag 017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:$CIRCLE_SHA1-arm64 honeycombio/supervised-collector:$CIRCLE_SHA1-arm64
+            docker push honeycombio/supervised-collector:$CIRCLE_SHA1-arm64
+            docker pull 017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:$CIRCLE_SHA1-amd64
+            docker tag 017118846235.dkr.ecr.us-east-1.amazonaws.com/supervised-collector:$CIRCLE_SHA1-amd64 honeycombio/supervised-collector:$CIRCLE_SHA1-amd64
+            docker push honeycombio/supervised-collector:$CIRCLE_SHA1-amd64
       - run:
           name: Login to Docker Hub
           command: echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
       - run:
-          name: Push Multi-Arch Docker Image
+          name: Build and Push Multi-Arch Docker Image
           command: |
-            docker buildx build \
-              --platform linux/amd64,linux/arm64 \
-              --sbom=true \
-              --provenance=true \
-              --push \
+            docker buildx imagetools create \
```

Comment:
Does this need a `--push`?

Reply:
no, imagetools pushes automatically unless you specify `--dry-run`